app/controllers/cards_controller.rb

```diff
@@ -1,6 +1,8 @@
+require 'super_memo_2'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.